website/docs/integrations/mixpanel.mdx

```diff
@@ -0,0 +1,206 @@
+)
+```
+</TabItem>
+
+<TabItem value="go" label="Go">
+```go
+client := configcat.NewCustomClient(configcat.Config{SDKKey: "#YOUR-SDK-KEY#",
+    Hooks: &configcat.Hooks{OnFlagEvaluated: func(details *EvaluationDetails) {
+        err := mp.Track(ctx, []*mixpanel.Event{
+            mp.NewEvent("$experiment_started", details.Data.User.(*UserData).Identifier, map[string]any{
+                "Experiment name": details.Data.Key,
+                "Variant name": details.Value,
+                "Variation ID": details.Data.VariationID,
+            }),
+        })
+    }}})
+```
+</TabItem>
+
+<TabItem value="java" label="Java">
+```java
+ConfigCatClient client = ConfigCatClient.get("#YOUR-SDK-KEY#", options -> {
+    options.hooks().addOnFlagEvaluated(details -> {
+        JSONObject props = new JSONObject();
+        props.put("Experiment name", details.getKey());
+        props.put("Variant name", details.getValue());
+        props.put("Variation ID", details.getVariationId());
+        JSONObject event = messageBuilder.event(details.getUser().getIdentifier(), "$experiment_started", props);
+
+        ClientDelivery delivery = new ClientDelivery();
+        delivery.addMessage(event);
+        mixpanel.deliver(delivery);
+    });
+});
+```
+</TabItem>
+
+<TabItem value="android" label="Android">
+```java
+ConfigCatClient client = ConfigCatClient.get("#YOUR-SDK-KEY#", options -> {
+    options.hooks().addOnFlagEvaluated(details -> {
+        JSONObject props = new JSONObject();
+        props.put("Experiment name", details.getKey());
+        props.put("Variant name", details.getValue());
+        props.put("Variation ID", details.getVariationId());
+        mp.track("$experiment_started", props);
+    });
+});
+```
+</TabItem>
+
+<TabItem value="ios" label="Swift (iOS)">
+```swift
+let client = ConfigCatClient.get(sdkKey: "#YOUR-SDK-KEY#") { options in
+    options.hooks.addOnFlagEvaluated { details in
+        Mixpanel.mainInstance().track(event:"$experiment_started", properties: [
+            "Experiment name": details.key,
+            "Variant name": details.value,
+            "Variation ID": details.variationId,
+        ])
+    }
+}
+```
+</TabItem>
+
+</Tabs>
+
+:::note
+If you are using Text feature flags with long text values (e.g. JSON values) that makes no sense to send into Mixpanel as a Variant, you can send the `variationId` as the `Variant name` to Mixpanel instead of the `value`.
+The `variationId` is a hashed version of the value and you can see the `variationId` on the ConfigCat Dashboard after turning on the _Show VariationIDs to support A/B testing_ 
+<a href="https://app.configcat.com/product/preferences" target="_blank" rel="noopener noreferrer">Product preference</a>.
+:::
+
+4. Deploy your application and wait for feature flag evaluations to happen so Experiments in Mixpanel could be populated.
+
+### Usage with Experiments
+Check your Experiments in Mixpanel. 
+TODO image, if we have access to the Experiments feature.
```

Comment:
TODO

Reply:
Yepp, i'm waiting for the Experiments feature to be unlocked in our account so I can create the screenshots. I already put the do not merge yet tag to the PR. I hope i can fix this soon.